### DISCUSSION TITLE
How to handle ChatGPT API rate limit with parallelGPT

### DISCUSSION THREAD

**Discussion by kivwil** [AntlerVC/firetable#1428] [Support / Q&A] [answered]
I use parallelGPT for bulk queries to openAI ChatGPT. Everything is fine except, that the script cannot handle the API rate limitation!
Is there a way to automatically have a kind of delay mechanism?

**Comment by Chadha93** [comment, marked as answer]
To handle the ChatGPT API rate limit with parallelGPT, you can implement a queue system that manages the requests and ensures that you stay within the rate limit.
